meson.build

```diff
@@ -29,6 +30,10 @@ realtime = cc.find_library('rt')
 wayland_client = dependency('wayland-client')
 wayland_protos = dependency('wayland-protocols', version: '>=1.14')
 
+if not cc.check_header('sys/timerfd.h', dependencies: epoll)
```

Comment:
Maybe check for `sys/epoll.h` instead? `dependencies: epoll` is probably unnecessary.

Is `cc.ahs_function('epoll')` not working?

Reply:
Ah, or are you checking for `sys/timerfd.h` because that's the only reason why we need `epoll`? Makes sense then.

We should probably migrate to a simple signal handler that writes to a pipe instead, but that's out of scope for this PR.